assets/dev/js/frontend/utils/anchor-scroll-margin.js

```diff
@@ -0,0 +1,153 @@
+module.exports = elementorModules.ViewModule.extend( {
+	getDefaultSettings() {
+		return {
+			selectors: {
+				links: '.elementor-element a[href*="#"]',
+				stickyElements: '.elementor-element.elementor-sticky',
+			},
+		};
+	},
+
+	onInit() {
+		const that = this;
+
+		this.observeStickyElements( () => {
```

Comment:
What happens when there are no sticky elements?

Do we only need to initialize this code when there are sticky elements?

Reply:
I have put that check in the functions below @hein-obox

Reply:
@Batsirai-muchareva 
Are those functions triggered if there are no sticky elements as well?